h2o-dist/index.html

```diff
@@ -367,42 +370,56 @@
         readBuildInfoJsonFile();
       }
     </script>
+    <link rel="stylesheet" type="text/css" href="static/css/main.css">
+    <link rel="stylesheet" type="text/css" href="static/css/bleeding-edge.css">
+    <link rel="stylesheet" href="https://use.typekit.net/gbq5zcz.css">
```

Comment:
Should this be in ./static?

Reply:
Typically, typekit stylesheets are linked to directly to typekit

Reply:
Ok, I will put this in the static folder